packages/smart/deploy/024_approve_market_factories.ts

```diff
@@ -11,7 +11,7 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
   const masterChefDeploy = await deployments.get("MasterChef");
   const masterChef = MasterChef__factory.connect(masterChefDeploy.address, signer);
 
-  const rewardsPerMarket = BigNumber.from(10).pow(18).mul(95);
+  const rewardsPerMarket = BigNumber.from(10).pow(18).mul(195);
```

Comment:
Maybe throw this magic number into a named constant? I can't tell if this is a typo or intentional.